app/models/rule_identifier.rb

```diff
@@ -7,7 +7,10 @@ class RuleIdentifier < ApplicationRecord
   validates :label, presence: true
   validates :system, presence: true
 
-  def self.from_oscap_rule(oscap_rule)
-    new(oscap_rule.identifier) if oscap_rule.identifier.dig(:label)
+  def self.from_openscap_parser(op_rule_identifier)
+    if op_rule_identifier.label # rubocop:disable Style/GuardClause
```

Comment:
Why disabling this instead of `return if op_rule_identifier_label.blank?`?

Reply:
rubocop gets confused here. The contents of the block, while one line, must wrap due to the line length requirement. When I use a guard clause, I need to wrap the one line across two or three, and rubocop then complains: `Style/MultilineIfModifier: Favor a normal if-statement over a modifier clause in a multiline statement.`. :man_shrugging: